config/label.go

```diff
@@ -26,10 +30,27 @@ func NewLabelConfig(APIAddress string) *LabelConfig {
 	}
 }
 
+// NewKubeLabelConfig gets a new LabelConfig for Kubernetes
+func NewKubeLabelConfig(APIAddress string, KubeConfig string, KubeNamespace string) *LabelConfig {
+	return &LabelConfig{
+		APIAddress:    APIAddress,
+		KubeConfig:    KubeConfig,
+		KubeNamespace: KubeNamespace,
+	}
+}
+
 // GetApps retrieves task config from the server using the API, and then gets scaling parameters from labels using MicroBadger
 func (l *LabelConfig) GetApps(userID string) (tasks []*demand.Task, maxContainers int, err error) {
 	tasks, maxContainers, err = api.GetApps(l.APIAddress, userID)
 	for _, task := range tasks {
```

Comment:
Yes the user downloads a k8s yaml file from the MiaB UI that has all the k8s objects. This file is where the image is specified.

The user can edit the yaml to change the image. Or change the app name in the MiaB UI to scale a different k8s deployment. The Marathon integration works in a similar way.